README.md

```diff
@@ -1,166 +1,167 @@
 
-:rocket: _Noch keine Erfahrung mit Docker? Gar kein Problem, weiter unten findest du eine [Anleitung für Einsteiger\_innen](#anleitung-für-einsteiger_innen-rocket)!_
+:rocket: _No experience with Docker yet? No worries, find a [Beginner’s guide](#beginners-guide-rocket) below!_
 
-__Ganz kurz, welchen Zweck erfüllt dieses Docker-Setup?__
+__What does the Docker Setup Exactly?__
 
-1. Docker stellt dir und deinem Team für jedes eurer REDAXO-Projekte eine __Serverumgebung (Apache, PHP, MySQL)__ bereit. Das funktioniert so ähnlich wie eine Virtuelle Maschine, benötigt aber viel weniger Ressourcen. Die Serverumgebung kannst du beliebig anpassen und jederzeit verwerfen, ohne dass Daten verloren gehen.
-2. Falls gewollt, kann Docker in der Serverumgebung eigenständig ein __frisches REDAXO installieren__.
-3. Noch besser: Docker kann sogar komplette __Demo-Websites installieren__, z. B. die beliebte [Basisdemo](https://github.com/FriendsOfREDAXO/demo_base) oder die [Community-Demo](https://github.com/FriendsOfREDAXO/demo_community). Damit kannst du jederzeit ohne besonderen Aufwand REDAXO-Features ausprobieren.
+1. Docker provides you and your team with a __Server environment (Apache, PHP, MySQL)__ for each of your REDAXO projects. It works much like a virtual machine, but uses much less resources. You can customize the server environment and discard it at any time without losing data.
+2. If desired, Docker can independently install a __fresh REDAXO in the server environment.__
+3. Much better: Docker can install the __Demo-Websites__, for example the [Basisdemo](https://github.com/FriendsOfREDAXO/demo_base) or the [Community-Demo](https://github.com/FriendsOfREDAXO/demo_community). You can try the REDAXO features anytime without special requirements.
 
-__Für wen ist sowas sinnvoll?__
+__Is it usefull for you?__
 
-* Für alle, die intensiv mit REDAXO arbeiten und __mehrere Projekte__ betreuen. Mit Docker kannst du jedem Projekt die passende Serverumgebung mitgeben, und die Auto-Installation ermöglicht dir, ohne besonderen Aufwand frische REDAXOs samt Demo-Websites zu generieren, an denen du Funktionen testen und entwickeln kannst.
-* Für Teams, denn sie erhalten dadurch eine __einheitliche Serverumgebung__ und sparen die Zeit, ihre Systeme jeweils manuell einrichten und pflegen zu müssen.
-* Für alle, die __komplexe Anwendungen__ entwickeln: Falls dein REDAXO eine Elasticsearch einsetzen möchte, einen externen Mailserver benötigt, Daten bei S3 auslagert oder sonstige Dienste benutzt, kannst du die Umgebung mit verschiedenen Docker-Containern abbilden.
+* For all those who work intensively with REDAXO and supervise __several projects__. With Docker, you can give any project the right server environment, and auto-installation allows you to easily generate fresh REDAXOs and demo sites to test and develop features.
+* For teams, because they get a __unified server environment__ and save the time to manually set up and maintain their systems.
```

Comment:
```suggestion
* Teams, because they are provided with a __unified server environment__ and save the time to manually set up and maintain their systems.
```